assets/css/_diversion_page.scss

```diff
@@ -9,10 +9,6 @@
     "map" 80vh
     / 1fr;
 
-  &__header {
-    grid-area: heading;
-  }
-
```

Comment:
question/issue: why was this removed?

Reply:
Seems like the intention was to get the "header" area to not be behind the panel?

This does that, but only by accident. Instead, the grid should be changed so that you can target the specific area to put the header in.

Reply:
OH, I heartily agree. I find the current grid layout super confusing, given that the header is tucked behind the panel, but also agree I shouldn't just torch it for this work. That said, we decided in slack to go ahead and merge, and then pair today to sort the grid out